docker-compose-dev.yml

```diff
@@ -37,15 +37,14 @@ services:
       - redis-data-dev:/data
       - ./data/redis/conf/redis.conf.template:/usr/local/etc/redis/redis.conf.template
       - ./data/certbot/conf/:/etc/letsencrypt
-    command: /bin/sh -c "sed -e 's/$$HOSTNAME/redis.${DOMAIN_NAME:-xlab.blindside-dev.com}/' /usr/local/etc/redis/redis.conf.template > /usr/local/etc/redis/redis.conf && exec redis-server --appendonly yes"
+    command: /bin/sh -c "sed -e 's/$$HOSTNAME/redis.${URL_HOST:-xlab.blindside-dev.com}/' /usr/local/etc/redis/redis.conf.template > /usr/local/etc/redis/redis.conf && exec redis-server --appendonly yes"
 
   certbot:
-    image: certbot/certbot
+    image: certbot/dns-route53
```

Comment:
This cannot be AWS specific for this script as it is used for many other deployments. In all case, we externalize the image with a config variable but default to `certbot/certbot`